Bottom: 91797e481875d22896eacaac9d667bd9ed68c949
Top:    afadb71fa3b9a675764b8c759d01ee62c27f2bac
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-30 11:20:32 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
